### ISSUE TITLE
Cannot use autoupdater without administrator privileges

### ISSUE THREAD

**Issue by mastergberry** [electron-userland/electron-builder#1129]
* **Version**: 11.4.1

* **Target**: Windows x64

Using the auto updater in our build. It is failing to install after calling `autoUpdater.quitAndInstall();` though because the application is installed via `"perMachine": true` and as a result needs administrator privileges to update.

Is there just some way to make it so every single time the application is run it automatically starts off with UAC already? This would be the easiest thing for me to fix the problem as I will need UAC also for other things.

**Comment by develar** [comment]
Hmm... Why Windows doesn't show UAC windows? To investigate. Installer can automatically request UAC.

**Comment by mastergberry** [comment]
Forgot to put the relevant part of the stack trace

```[2017-01-17 09:55:58:0449] [error] Error: Error: spawn UNKNOWN
    at exports._errnoException (util.js:1026:11)
    at ChildProcess.spawn (internal/child_process.js:313:11)
    at exports.spawn (child_process.js:392:9)```

From other issues in the repo I saw that it seems to be a UAC permission error.

**Comment by mastergberry** [comment]
Confirmed when starting the application with Administrator privileges it was able to auto update properly.

**Comment by develar** [comment]
Working on fix.

**Comment by develar** [comment]
Fixed in `electron-updater` 1.2.0. **Requires `electron-builder` 11.6.0.**
